src/lightning-stubs/toastContainer/toastContainer.js

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+export const ToastContainer = {
+    instance: jest.fn(() => ({})),
+};
```

Comment:
```suggestion
import { LightningElement, api } from 'lwc';

export default class ToastContainer extends LightningElement {
    @api containerPosition;
    @api maxToasts;
    @api toastPosition;
    @api close() {}
}
```

Reply:
@Templarian happy to accept these changes but unless the [lightning/toastContainer documentation](https://developer.salesforce.com/docs/component-library/bundle/lightning-toast-container/documentation) is incorrect we also need a static `instance()` method, no?

Reply:
@pozil Oh good catch. Quickly copied our internal mock that is missing it also. Updated the code suggestion above to include it.